activation/handler.go

```diff
@@ -121,23 +122,38 @@ func (h *Handler) ProcessAtx(ctx context.Context, atx *types.VerifiedActivationT
 	if existingATX != nil { // Already processed
 		return nil
 	}
-	epoch := atx.PublishEpoch()
 	h.log.WithContext(ctx).With().Info("processing atx",
-		atx.ID(),
-		epoch,
+		log.Stringer("atx_id", atx.ID()),
```

Comment:
That's true, I updated some log lines because I was bug hunting and I was confused by log lines which had multiple fields with the same name but different values, i.e. 3 `node_id` fields, one of which was different from the others.

I will revert some of the log changes.